monitoring/ceph-mixin/dashboards/ceph-nvmeof.libsonnet

```diff
@@ -119,191 +119,164 @@ local g = import 'grafonnet/grafana.libsonnet';
+          format='time_series',
+          instant=true,
+          legendFormat="Down",
+          range=false,
+          datasource='$datasource',
+        )
       )
-    )
-    .addOverrides([
-      {
-        matcher: { id: 'byName', options: 'Down' },
-        properties: [
-          {
-            id: 'color',
-            value: {
-              fixedColor: 'red',
-              mode: 'fixed',
+      .addOverrides([
+        {
+          matcher: { id: 'byName', options: 'Down' },
+          properties: [
+            {
+              id: 'color',
+              value: { 
+                fixedColor: 'red', 
+                mode: 'fixed' 
+              },
             },
-          },
-        ],
-      },
-      {
-        matcher: { id: 'byName', options: 'Total' },
-        properties: [
-          {
-            id: 'color',
-            value: {
-              fixedColor: '#a7a38b',
-              mode: 'fixed',
+          ],
+        },
+        {
+          matcher: { id: 'byName', options: 'Total' },
+          properties: [
+            {
+              id: 'color',
+              value: { 
+                fixedColor: '#a7a38b', 
+                mode: 'fixed' 
+              },
             },
-          },
-        ],
-      },
-      {
-        matcher: { id: 'byName', options: 'Available' },
-        properties: [
-          {
-            id: 'color',
-            value: {
-              fixedColor: 'green',
-              mode: 'fixed',
+          ],
+        },
+        {
+          matcher: { id: 'byName', options: 'Available' },
+          properties: [
+            {
+              id: 'color',
+              value: { 
+                fixedColor: 'green', 
+                mode: 'fixed' 
+              },
             },
-          },
-        ],
-      },
-    ]),
+          ],
+        },
+      ]),
 
-    $.timeSeriesPanel(
-      title='Ceph Health NVMeoF WARNING',
-      description='Ceph healthchecks NVMeoF WARNINGs',
-      gridPosition={ x: 8, y: 1, w: 7, h: 8 },
-      lineInterpolation='linear',
-      lineWidth=1,
-      drawStyle='line',
-      axisPlacement='auto',
-      datasource='$datasource',
-      fillOpacity=5,
-      pointSize=5,
-      showPoints='auto',
-      unit='none',
-      displayMode='list',
-      showLegend=true,
-      placement='bottom',
-      tooltip={ mode: 'multi', sort: 'desc' },
-      stackingMode='normal',
-      spanNulls=false,
-      decimals=0,
-      thresholdsMode='absolute',
-      noValue='0',
-    ).addThresholds([
-      { color: 'green', value: null },
-    ])
-    .addTarget(
-      $.addTargetSchema(
-        expr="sum(ceph_health_detail{name='NVMEOF_GATEWAY_DOWN'})",
-        format='',
-        instant=false,
-        legendFormat='NVMEOF_GATEWAY_DOWN',
-        range=true,
+      $.timeSeriesPanel( 
+        title='Gateway WARNING Health States',
```

Comment:
We can include the healthy one's as well. Maybe call it `Gateway health status`?

Reply:
The graph shows total number of gateways in bad state (and shows daemons names and the state on hover). That's why I'm not sure how we can include the healthy ones here. If we show all states, then unhealthy states would not be highlighted. 

<img width="1193" height="883" alt="Screenshot 2025-10-08 at 1 12 28 PM" src="https://github.com/user-attachments/assets/1f835b1d-31be-4354-b3ee-b0e2bdc0b402" />

Reply:
The thing is even though description of pane says Warning health states -> The stopped or error are danger/error state.  So the Panel name looks confusing.

I think we don't need graph for representation the state. Its a point in time data, so single stat should be good. I find it confusing to read personally.